### PR TITLE
Update llama2.md

### PR DESCRIPTION
**Pull request by philschmid**
Updates inference suggestion to new instance types

### CODE REVIEW

**Review by pcuenca — APPROVED** [review]

File: llama2.md [hunk 1]
```diff
@@ -147,7 +147,7 @@ You can try out Text Generation Inference on your own infrastructure, or you can
 
 - For 7B models, we advise you to select "GPU [medium] - 1x Nvidia A10G".
 - For 13B models, we advise you to select "GPU [xlarge] - 1x Nvidia A100".
-- For 70B models, we advise you to select "GPU [xxxlarge] - 8x Nvidia A100".
+- For 70B models, we advise you to select "GPU [2xlarge] - 2x Nvidia A100" wit quantization `bitsandbytes` enabled or "GPU [4xlarge] - 4x Nvidia A100"
```
[the remark this answers is not in the document]
Comment (philschmid) [reply llama2.md:150]: Its done by TGI and in the UI you only select `bitsandbytes`, see 
image

[the remark this answers is not in the document]
Comment (osanseviero) [reply llama2.md:150]: That's cool!

Comment (osanseviero) [inline llama2.md:150]:
```suggestion
- For 70B models, we advise you to select "GPU [2xlarge] - 2x Nvidia A100" with `bitsandbytes` quantization enabled or "GPU [4xlarge] - 4x Nvidia A100"
```

**Review by osanseviero — APPROVED** [review]
Cool, thanks! :fire:

**Pull request merged**